.ci/robot_framework/tests/tests_021_stress_ng.robot

```diff
@@ -0,0 +1,26 @@
+*** Settings ***
+Test Timeout    60 seconds
+
+Library    OperatingSystem
+Library    ../libs/TestUtils.py
+
+Resource   variables.robot
+
+*** Test Cases ***
```

Comment:
rename `.ci/robot_framework/tests/tests_021_stress_ng.robot` to `.ci/robot_framework/tests/tests_006_stress_ng.robot`